commands/http/client.go

```diff
@@ -145,6 +158,18 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 	}
 }
 
+func streamingRequest(url string, req cmds.Request) (cmds.Response, error) {
+	conn, httpRes, err := websocket.DefaultDialer.Dial(url, nil)
+	if err != nil { return nil, err }
+	wsio := NewWebsocketIO(conn)
```

Comment:
we should really just make a websocket (and unix domain socket) version of the commands lib api (ie support everything)